configure.ac

```diff
@@ -55,7 +55,7 @@ if test "$GCC" = yes; then
 -Wshadow -Wpointer-arith $cast_align -Wwrite-strings \
 -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes \
 -Wnested-externs $CFLAGS"
-    CXXFLAGS=" -g -W -Wall -Wpointer-arith $cast_align $wshadow -Wwrite-strings $CXXFLAGS"
+    CXXFLAGS=" -std=gnu++11 -g -W -Wall -Wpointer-arith $cast_align $wshadow -Wwrite-strings $CXXFLAGS"
```

Comment:
This is okay, but I would prefer a minimum level of this, while allowing higher standards if the compiler supports it... (I'm working on cmake which handles this easier than autotools this so don't worry about it)

Reply:
yeah just the code get so much nicer ... for the for () loops
I can rewrite it the old way ... and remove if you want

Reply:
No, I'm saying that if C++14 is available I want to use that instead of C++11. 

I'm not sure how far back we want to support C++ standards, maybe the next release should require C++11, and C++17 for the one after that?

Reply:
My vote would be c++ 14 for the next release, and 17 after that.   For what it's worth, I've seen several areas of code that could profit from string_view in C++ 17.